_data/tutorial-groups.yml

```diff
@@ -0,0 +1,27 @@
+- id: networking
```

Comment:
We are getting enough networking pages that we might consider splitting them into "Basic Networking" for things like connecting to the Internet and "Advanced Networking" for things like NFS and MQTT.